src/pages/home/_hero.tsx

```diff
@@ -0,0 +1,226 @@
+import { Localize, localize } from 'components/localization'
+
+const query = graphql`
+    query {
+        hero_background: file(relativePath: { eq: "home/hero_bg.png" }) {
+            ...homePageHeroFadeIn
+        }
+        hero_platform1: file(relativePath: { eq: "home/hero_platform1.png" }) {
+            ...fadeIn
+        }
+        hero_platform2: file(relativePath: { eq: "home/hero_platform2.png" }) {
+            ...fadeIn
+        }
+        hero_platform3: file(relativePath: { eq: "home/hero_platform3.png" }) {
+            ...fadeIn
+        }
+        hero_platform4: file(relativePath: { eq: "home/hero_platform4.png" }) {
+            ...fadeIn
+        }
+    }
+`
+type HeroProps = {
+    is_ppc: boolean
+}
+
+const contents = [
+    <Localize key={0} translate_text="Tight spreads" />,
+    <Localize key={1} translate_text="Sharp prices" />,
+    <Localize key={2} translate_text="24x7 trading" />,
+    <Localize key={3} translate_text="100+ tradeable assets" />,
+    <Localize key={4} translate_text="20+ years of experience" />,
+]
+
+const contents_ppc = [
+    <Localize key={3} translate_text="Tight spreads" />,
+    <Localize key={2} translate_text="Sharp prices" />,
+    <Localize key={1} translate_text="100+ tradeable assets" />,
+    <Localize key={0} translate_text="20+ years of experience" />,
+]
+
+const StyledContainer = styled(Container)`
+    width: 100%;
+    max-width: 100%;
+    margin: 0 auto;
+    padding: 0 12rem;
+
+    @media ${device.laptopM} {
+        padding: 0 8rem;
+    }
+
+    @media ${device.tabletL} {
+        align-items: center;
+        max-width: 588px;
+        padding: 0 16px;
+    }
+`
+const StyledHeroContainer = styled(Flex)`
+    max-width: 1200px;
+    margin: 0 auto;
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+        max-width: 100%;
+    }
+`
+const StyledHeaderWrapper = styled(Flex)`
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        justify-content: flex-start;
+    }
+`
+const HeroWrapper = styled.section`
+    width: 100%;
+    padding: 80px 0;
+    background: linear-gradient(241.35deg, #122434 12.86%, #060c11 85.61%, #060c11 85.61%);
+    position: relative;
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        justify-content: flex-start;
+        padding: 40px 0 46px;
+    }
+`
+
+const HeroButton = styled(LinkButton)`
+    padding: 17px 24px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: fit-content;
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+        font-size: 20px;
+        line-height: 30px;
+    }
+    @media ${device.mobileS} {
+        font-size: 18px;
+        padding: 12px 20px;
+    }
+`
+const StyledHeader = styled(Header)`
+    font-size: 80px;
+    line-height: 100px;
+
+    @media ${device.tabletL} {
+        font-size: 40px;
+        line-height: 50px;
+        margin-right: 8px;
+        width: unset;
+    }
+`
```

Comment:
This seems to be an inconsistent font type for desktop/mobile based on the size & line height. Could you check with the designer?